File: app/src/main/java/com/benny/openlauncher/activity/AutoFinishActivity.java
package com.benny.openlauncher.activity;

import android.app.Activity;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;

public class AutoFinishActivity extends Activity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        finish();
    }

    public static void start(Context context) {
        context.startActivity(new Intent(context, AutoFinishActivity.class));
    }
}

File: app/src/main/java/com/benny/openlauncher/activity/AutoFinishActivity.kt
package com.benny.openlauncher.activity

import android.app.Activity
import android.content.Context
import android.content.Intent
import android.os.Bundle

//For kustom support
class AutoFinishActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        finish()
        super.onCreate(savedInstanceState)
    }

    companion object {

        fun start(c: Context) {
            c.startActivity(Intent(c, AutoFinishActivity::class.java))
        }
    }
}
